File: 2.0/python/texts/getTextAutoReply.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.texts.getTextAutoReply(
    id=379506003,
    # return only specific fields
    fields='keyword,message'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/texts/getTextAutoReply.php
<?php

class ApiClientSample{

    public static function main(){
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getTextAutoReply();
        $request->getOperationConfig()->setPathParameters(array("id" => 12345));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();

